JoinRpg.Services.Impl/ProjectService.cs

```diff
@@ -17,9 +17,10 @@ namespace JoinRpg.Services.Impl
   [UsedImplicitly]
   public class ProjectService : DbServiceImplBase, IProjectService
```

Comment:
While we are here: please mark as TODO to move Character operations into it own service.